Src/CSharpier.Tests/FormattingTests/TestFiles/InitializerExpressions.cst

```diff
@@ -141,6 +141,27 @@ class ClassName
             new Thing { One = 1 },
             Two = 2
         };
+
+        var conditionalsAndInvocations = new List<SomeObject>
+        {
+            this.CallSomeMethod________________________________________()
+                .CallSomeMethod________________________________________()
+              ? one
+              : two,
+            SomeOtherMethod(),
+            SomethingElse
+                .CallSomeMethod________________________________________()
+                .CallSomeMethod________________________________________()
+              ? one
+              : two,
+            SomeOtherMethod(),
+            this.CallSomeMethod________________________________________()
+                .CallSomeMethod________________________________________(),
+            someLongCondition___________________________________
+            && someLongCondition___________________________________
```

Comment:
In this case if it were indented, then it would line up with the ?
Maybe #3 below is the way to go? I think there might be some cases where binary operators do get indented.
```c#
someLongCondition___________________________________
&& someLongCondition___________________________________
    ? one
    : two

someLongCondition___________________________________
    && someLongCondition___________________________________
    ? one
    : two

someLongCondition___________________________________
    && someLongCondition___________________________________
  ? one
  : two